src/server.c

```diff
@@ -5813,8 +5813,8 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                                     "offset=%lld,lag=%ld,type=%s\r\n",
                                     replica_id, replica_ip, replica->replica_listening_port, state,
                                     replica->repl_ack_off, lag,
-                                    replica->flag.repl_rdb_conn                        ? "rdb-conn"
-                                    : replica->repl_state == REPLICA_STATE_BG_RDB_LOAD ? "main-conn"
+                                    replica->flag.repl_rdb_chan                        ? "rdb-chan"
+                                    : replica->repl_state == REPLICA_STATE_BG_RDB_LOAD ? "main-chan"
                                                                                        : "replica");
```

Comment:
(INFO field per replica) @valkey-io/core-team Terminology is OK?

I see some new terminology here "chan" (new abbreviation) "main-chan", "rdb-chan".

Maybe use full word "channel"?

Reply:
I would use the full word, "rdb-channel".

Reply:
Yeah I think "channel" is better.

Reply:
👍